client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/sounds/add/telecom-telephony-alias-configuration-contactCenterSolution-sounds-add.controller.js

```diff
@@ -0,0 +1,52 @@
+angular.module('managerApp').controller('TelecomTelephonyAliasConfigurationContactCenterSolutionSoundsAddCtrl', class TelecomTelephonyAliasConfigurationContactCenterSolutionSoundsAddCtrl {
+  constructor(
+    $q, $stateParams, $translate, $uibModalInstance,
```

Comment:
-`$q` and `$uibModalInstance`

Reply:
Ok for `$q` but `$uibModalInstance` is used into html